controllers/cloud.redhat.com/providers/serviceaccount/provider.go

```diff
@@ -20,5 +20,5 @@ func GetServiceAccount(c *p.Provider) (p.ClowderProvider, error) {
 }
 
 func init() {
-	p.ProvidersRegistration.Register(GetServiceAccount, 1, ProvName)
+	p.ProvidersRegistration.Register(GetServiceAccount, 97, ProvName)
```

Comment:
What's the reason for registering SA's towards the end?

Reply:
Only because we need to add it to most deployments, and so we want to wait until as many providers have created their resources as possible,